tests/OptimizeTest.elm

```diff
@@ -16,11 +17,29 @@ import Stage.Optimize
 import Test exposing (Test, describe, test, todo)
 
 
+typed : Typed.Expr -> Typed.LocatedExpr
+typed expr =
+    Located.located
+        -- position do not matters in optimize
+        { start = { row = 0, col = 0 }, end = { row = 0, col = 0 } }
+        expr
+
+
+typedInt : Int -> Typed.LocatedExpr
+typedInt int =
+    typed ( Literal (Int int), Type.Int )
```

Comment:
making a note here to later try extracting all these helpers into some separate module that can be used by all those test files that define it themselves now...